util/version/src/lib.rs

```diff
@@ -63,7 +63,7 @@ pub fn version_data() -> Bytes {
 		(env!("CARGO_PKG_VERSION_MINOR").parse::<u32>().expect("Environment variables are known to be valid; qed") << 8) +
 		env!("CARGO_PKG_VERSION_PATCH").parse::<u32>().expect("Environment variables are known to be valid; qed");
 	s.append(&v);
-	s.append(&"Parity-Ethereum");
+	s.append(&"Open-Ethereum");
```

Comment:
and here as well

Reply:
I wonder if there are implications of doing this outside the code itself, I'm thinking of things like trackers and whatnot? /cc @joshua-mir ?

Reply:
https://ethernodes.org/ perhaps, although I don't know how they identify clients